playbooks/kubernetes.md

```diff
@@ -71,8 +85,35 @@ To diagnose application failures, navigate to the Kubernetes dashboard and searc
 search box at the top of the dashboard to filter all Kubernetes domain objects by application name. Once you have a
 view on your application's resources, look for any Pods in an "Error" or "CrashLoopBackoff" state, then _Delete_
 these pods by either using the three-dots "actions" menu to the right of the pod's CPU / Memory graphs, or click
-through to the Pod view and use the "Delete" menu with the trash can at the top of the page.
+through to the Pod view and use the "Delete" menu with the trash can at the top of the page, or via the `kubectl` CLI: `kubectl delete {pod-name}`
 
 Alternatively, you can (in the application's git checkout, assuming you have the Hokusai CLI installed) run the
-command `hokusai [staging|production] refresh` but be aware this is a _hard_ restart of the application and may
-result in brief downtime as it will delete _all_ running pods simultaneously.
+command `hokusai [staging|production] refresh`
+
+#### When things go wrong (cluster level)
+
+Kubernetes cluster nodes may experience issues with their underlying system (too many open file handles, full disk, memory, etc) and require system-level debugging, or occasionally need to be re-provisioned.
+
+To inspect a node, first find its FQDN, which is the private DNS name assigned by AWS EC2 autoscaling.
+
+To view all nodes for a given cluster, run `kubectl get nodes`.  Another useful command, if you wish to see pods along with the node on which they are scheduled / running is `kubectl get pods -o wide`
```

Comment:
You can also get the the FQDN of a pod's node via `hokusai [staging|production] status`, for example:

```
dl@phoenix:~/src/force (healthier-healthchecks)
$ hokusai staging status

Resources
===========
NAME                              DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE       CONTAINERS   IMAGES                                                                                        SELECTOR
deployment.extensions/force-web   3         3         3            3           91d       force-web    585031190124.dkr.ecr.us-east-1.amazonaws.com/force:6b1a62fc782b61d456f92e72225f53561c7ef2cc   app=force,component=web,layer=application

NAME                TYPE           CLUSTER-IP      EXTERNAL-IP                                                               PORT(S)                      AGE       SELECTOR
service/force-web   LoadBalancer   100.68.109.20   a099d3c23bb1e11e891cf1273498b4c6-1023438724.us-east-1.elb.amazonaws.com   443:31693/TCP,80:30170/TCP   91d       app=force,component=web,layer=application

Pods
===========
NAME                         READY     STATUS    RESTARTS   AGE       IP               NODE
force-web-749f46cd79-c2tsv   1/1       Running   0          10h       100.123.47.36    ip-10-0-1-235.ec2.internal
force-web-749f46cd79-gf5m7   1/1       Running   0          11h       100.120.48.176   ip-10-0-2-161.ec2.internal
force-web-749f46cd79-t6p82   1/1       Running   0          11h       100.102.237.62   ip-10-0-1-233.ec2.internal
```

Might be a bit more useful if trying to quickly triangulate which hosts could be problematic for a given service. Food for thought!

Reply:
Added a note for this as well